content/en/logs/log_collection/docker.md

```diff
@@ -103,11 +103,13 @@ The source and service values can be overridden with Autodiscovery as described
 
 * Logs coming from container `Stderr` have a default status of `Error`.
 
+* Refer to the [journald integration][1] documentation for details regarding the setup for containerized environments if using the journald logging driver instead of Docker's default json-file logging driver.
```

Comment:
```suggestion
* If using the journald logging driver instead of Docker's default json-file logging driver, see the [journald integration][1] documentation for details regarding the setup for containerized environments.
```